tls/s2n_resume.c

```diff
@@ -228,27 +228,53 @@ static int s2n_client_serialize_resumption_state(struct s2n_connection *conn, st
     return 0;
 }
 
-static S2N_RESULT s2n_tls12_client_deserialize_session_state(struct s2n_connection *conn, struct s2n_stuffer *from)
+static S2N_RESULT s2n_tls12_client_deserialize_session_state(struct s2n_connection *conn,
```

Comment:
Shouldn't you also do this for the tls13 deserialization?

Reply:
Also do what? I added the ticket as an input here, but the TLS1.3 deserialization already takes the ticket as an input. It just calls it the psk identity because that's what we use it for :)